.github/actions/package/action.yml

```diff
@@ -103,3 +103,10 @@ runs:
         name: |
           android-build
           ios-build
+
+    ### ONLY FOR TESTING INSIDE THE PR -- See .github/workflows/release.yml for the final destination ##
+    - name: Generate Sponsors
+      uses: JamesIves/github-sponsors-readme-action@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
As this is a composite action not a workflow definition there is no access to secrets. It would need to be declared as an in the `inputs` block on line 4, and then referenced as `inputs.GITHUB_TOKEN`.

But given that you're just trying to test it out maybe just add it to `workflows/build.yml` after the package step that references this file rather than do all that to have it work in this file?

Reply:
Thank you! Updated!